product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -4,9 +4,47 @@ redirects:
 - /biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint/
 ---
 
+
+
+AWS VPC endpoint (AWS Private Link) service is a network interface that securely connects a private IP address from your AWS VPC to an external service. You grant access only to a single cluster instead of the entire BigAnimal resource VPC, thus ensuring maximum network isolation. 
+
+For more information, see [VPC endpoint services (AWS PrivateLink)](https://docs.aws.amazon.com/vpc/latest/privatelink/endpoint-service.html).
+
+The way you create a private endpoint differs when you are using your own AWS account as opposed to using BigAnimal's cloud account.
+
+## Using BigAnimal's cloud account
+
+When using BigAnimal's cloud account, you provide BigAnimal with your AWS account ID when creating a cluster. BigAnimal, in turn, provides you with a AWS service name, which you can use to connect to your cluster privately.
+
+1. When creating your cluster, on the **Cluster Settings** tab, in the **Network Connectivity** section:
+   1. Select **Private**.
+
+   1. Enter your application's AWS account ID.
+
+1. After the cluster is created, go to its cluster details to see the corresponding endpoint service name. You need the service name while creating a VPC endpoint.
+
+1. Create a VPC endpoint in the client's VPC. The steps for creating a VPC endpoint in the client's VPC are the same  whether you are using BigAnimal's cloud or your own. The steps are available [here](#step-2-create-a-vpc-endpoint-in-the-clients-vpc).
+
+1. In your application's AWS account, select **VPC** and then select **Endpoints**. Select the endpoint you created previously and use the service name provided in the details section in BigAnimal to access your cluster.
+
+   ```shell     
+   # Replace the port 5432 below if that has been changed in the Big Animal DB Configuration
+   psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin -p 5432     
```

Comment:
this FQDN is not good, we don't have to use the AWS provided one, but the BigAnimal generated one, like
`PG-CLUSTER-ID.private.biganimal.io` (not really sure the final BAH domain is correct here, asking @nstott for confirmation 🙏 )